src/renderer/components/+workloads-replicasets/replica-set-menu.tsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
```

Comment:
Note: This component was extracted to separate file to avoid circular dependency